specutils/manipulation/utils.py

```diff
@@ -152,3 +152,42 @@ def excise_region(spectrum, region, exciser=linear_exciser):
     #
 
     return exciser(spectrum, region)
+
+
+def spectrum_from_model(model_input, spectrum):
```

Comment:
This needs to be added to `__all__` if it's intended for the public API

Reply:
👍